bullet_train-api/app/helpers/api/open_api_helper.rb

```diff
@@ -183,9 +183,11 @@ def description_for(model)
 
     private
 
-    def has_strong_parameters?(controller)
-      methods = controller.action_methods
-      methods.include?("create") || methods.include?("update")
+    def has_strong_parameters?(controller_name)
+      "#{controller_name}::StrongParameters".constantize
+      true
+    rescue NameError
+      false
```

Comment:
These changes make it so that this method actually does what it says. Previously it did something closer to `should_have_strong_parameters?` and that led to problems if `create` or `update` was defined but a `StrongParameters` module was not.